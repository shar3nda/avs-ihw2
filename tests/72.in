ly}<8Yh<S:.%t_	*8E=@ZdtU-4Lvy$5@L`Zk{5p,ISugTaxY|TD'3OGivIZ}wI~K<Y(Q%&N!\HYc