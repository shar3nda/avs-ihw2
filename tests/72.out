Lowercase: 385, uppercase: 391